src/Magento/FunctionalTestingFramework/DataGenerator/Handlers/SecretStorage/AwsSecretsManagerStorage.php

```diff
@@ -115,17 +115,18 @@ public function getEncryptedValue($key)
             $reValue = openssl_encrypt($value, parent::ENCRYPTION_ALGO, parent::$encodedKey, 0, parent::$iv);
             parent::$cachedSecretData[$key] = $reValue;
         } catch (AwsException $e) {
-            $error = $e->getAwsErrorCode();
+            $errMessage = "\nAWS Exception:\n" . $e->getAwsErrorMessage()
+                . "\nUnable to read value for key {$key} from AWS Secrets Manager\n";
+            print_r($errMessage);
```

Comment:
is there a way to get the aws exceptions to display in allure as well?

Reply:
> Few comments added, may need more elaborate documentation on local use, focusing on setting up .aws credentials file with IAM user access key id, aws_access_key_id, aws_secret_access_key and region

This is mentioned in section ### Setup MFTF to use AWS Secrets Manager